aria-practices.html

```diff
@@ -712,31 +712,30 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
     <section class="widget" id="combobox">
       <h3>Combo Box</h3>
       <p>
-        A <a href="#combobox" class="role-reference">combobox</a> is a widget made up of the combination of two distinct elements:
-        1) a single-line textbox, and 2) an associated pop-up element for helping users set the value of the textbox.
+        A <a href="#combobox" class="role-reference">combobox</a> is a widget that controls a pop-up element for helping users set the value of the combobox.
```

Comment:
The spelling of pop-up vs popup is inconsistent. I prefer popup (without a dash).

Reply:
Changed in https://github.com/w3c/aria-practices/pull/1255/commits/621b26c6c89c79c5219ae061b2736d0f87df8855